doc/mf6io/gwf/binaryoutput.tex

```diff
@@ -275,6 +275,7 @@ \subsubsection{Advanced Flow and Transport Packages}
 GWF/LAK & STAGE & Simulated lake stage  \\
 GWF/SFR & STAGE & Simulated stream reach stage  \\
 GWF/MAW & HEAD & Simulated well head  \\
+GWF/UZF & WATER CONTENT & Simulated unsaturated zone cell water content \\
```

Comment:
Isn't the header supposed to be WATER-CONTENT. This should be the text string written to the binary file.

Reply:
Great catch, fixed with f0b161b